src/firefly/java/edu/caltech/ipac/visualize/plot/WorldPt.java

```diff
@@ -56,16 +55,16 @@ public String serialize() {
     protected static WorldPt stringAryToWorldPt(String wpParts[]) {
         WorldPt retval= null;
         if (wpParts.length>=3) {
-            double lon= StringUtils.parseDouble(wpParts[0]);
-            double lat= StringUtils.parseDouble(wpParts[1]);
+            double lon= parseDouble(wpParts[0]);
+            double lat= parseDouble(wpParts[1]);
```

Comment:
It is doing the same as before.  I move the parse double to the base class Pt.java.  I am trying to remove the dependency on StringUtils to make the new GWT cross compile smaller.